src/classes/STG_Base_CTRL.cls

```diff
@@ -279,4 +279,19 @@ public with sharing class STG_Base_CTRL {
             return jobId;
         }
     }
+    
+    /*******************************************************************************************************
+    * @description Executes Batch to clean up data for Preferred Phone enforcement
+    * @return Id of Batch Job
+    */
+    @AuraEnabled
+    public static Id executePreferredPhoneCleanUpBatch() {
+        CON_Phone_BATCH contbatch = new CON_Phone_BATCH(null);
+        Id jobId = Database.executeBatch( contbatch , 200 );
+        if(jobId == null) {
+            throw new AuraHandledException('There was a problem starting the Batch.');
```

Comment:
Is this what we do for the preferred email batch job too? That's unfortunate if so (would prefer it to be a translated label but not going to stress about it too much if this was a copy/paste...)

Reply:
it was a copy past :)

Reply:
That said, we should look at turning this into a label. Doesn't have to be this go-round though.